### PR TITLE
Get rid of members

### PR DESCRIPTION
**Pull request by brian-avery**
This merges the developers and members teams on Istio. Output is attached.

[members_and_developers.yaml.txt](https://github.com/istio/community/files/6670892/members_and_developers.yaml.txt) -- this is the old version with separate groups for members and developers
[members.yaml.txt](https://github.com/istio/community/files/6670893/members.yaml.txt) -- this is the new version, which combines members and developers into a single group called `members`.

Fixes https://github.com/istio/community/issues/476

### CODE REVIEW

File: org/gen.go [hunk 1]
```diff
@@ -114,17 +113,16 @@ func strPointer(s string) *string {
...
 func convertConfig(cfg Organization) org.FullConfig {
 	allMembers := cfg.Members[:]
-	allMembers = append(allMembers, cfg.Developers...)
 	sort.Slice(allMembers, func(i, j int) bool { return strings.ToLower(allMembers[i]) < strings.ToLower(allMembers[j]) })
 
-	// Insert the developers team, which is handled separately
+	// Insert the members team, which is handled separately
 	closed := org.Closed
-	cfg.Teams["Developers"] = org.Team{
+	cfg.Teams["Members"] = org.Team{
```
Comment (howardjohn) [inline org/gen.go:120]: is creating a new team right? Does this new team have the correct permissions? Won't everyone get notified that they were removed from "Developers"?

Comment (brian-avery) [reply org/gen.go:120]: This makes the permissions the same between members and developers. Yes, it does mean that everyone in developers will get an email that they were removed from developers and added to members. Originally I was going to change everyone to be a developer, but the requirement from #476 was that the new term to be used was members. It didn't make sense to me to refer to them as members in one place and developers in another.

Comment (brian-avery) [reply org/gen.go:120]: So I guess the question here is what's the term we want to use? Developers? Members? Contributors? 

* If we use a different team than the two prior groups then it avoids confusion of prior understandings. 
* Developers may not make sense for, for example, docs writers and other contributors who don't actually write code.

Comment (howardjohn) [reply org/gen.go:120]: maybe we can just rename the team in the UI, then do this, to avoid churn

Comment (craigbox) [reply org/gen.go:120]: I like the sound of that

Comment (craigbox) [reply org/gen.go:120]: Could we disable sending e-mails while we do it?

Comment (brian-avery) [reply org/gen.go:120]: @craigbox Like the sound of what? Renaming? What term do we want to use?

Comment (craigbox) [reply org/gen.go:120]: renaming the teams in the UI, sorry.  I would like the end result to be "members" for the reason you spelled out above, but I don't mind how we get there, and agree with minimising confusion due to automated e-mails.

File: org/gen.go [hunk 2]
```diff
@@ -114,23 +113,24 @@ func strPointer(s string) *string {
...
 	}
 
+	triagePerm := github.Triage
 	istio := org.Config{
 		Metadata: org.Metadata{
-			Name:        strptr("Istio"),
-			Description: strptr("Connect, secure, control, and observe services."),
+			Name:                        strptr("Istio"),
+			Description:                 strptr("Connect, secure, control, and observe services."),
+			DefaultRepositoryPermission: &triagePerm,
```
Comment (brian-avery) [inline org/gen.go:133]: @howardjohn this should set the default org permissions to Triage

File: org/config_test.go [hunk 3]
```diff
@@ -108,19 +107,10 @@ func TestIstioOrg(t *testing.T) {
 	if !isSorted(cfg.Members) {
 		t.Errorf("members unsorted")
 	}
-	if !isSorted(cfg.Developers) {
-		t.Errorf("developers unsorted")
-	}
 	if !isSorted(cfg.Admins) {
 		t.Errorf("members unsorted")
```
Comment (craigbox) [inline org/config_test.go:111]: nit: this should say "admins unsorted'?

Comment (brian-avery) [reply org/config_test.go]: Good catch.

**Pull request merged**